site/docs/v1/tech/installation-guide/kubernetes/setup/minikube.adoc

```diff
@@ -146,9 +147,28 @@ To connect to your database from outside the cluster execute the following comma
     PGPASSWORD="$POSTGRES_PASSWORD" psql --host 127.0.0.1 -U postgres -d postgres -p 5432
 ----
 
-When you deploy FusionAuth, you will need to use the DNS name `pg-minikube-postgresql.default.svc.cluster.local` as seen above and the password that you set in the install command.
+When you deploy FusionAuth, you will need to use the DNS name `pg-minikube-postgresql.default.svc.cluster.local` as seen above and the password created from installing the helm chart.
+
+[source,helmtext,title=Export the Postgres password]
+----
+export POSTGRES_PASSWORD=$(kubectl get secret --namespace default pg-minikube-postgresql -o jsonpath="{.data.postgresql-password}" | base64 --decode)
+----
+
+Following the above command, if you did not set a password when installing my-release you can use the command below to display it.
+
+[source,helmtext, title=Display your password]
+----
+echo $POSTGRES_PASSWORD
+----
+
+You can also test your deployment by attempting to connect to the database following the instructions in the output.
```

Comment:
```suggestion
You can also test your deployment by attempting to connect to the database.
```

I don't know what 'instructions in the output' means? Are you saying that this command outputs instructions you then need to copy and paste?